tests/ui/consts/miri_unleashed/ptr_arith.rs

```diff
@@ -1,5 +1,5 @@
 // compile-flags: -Zunleash-the-miri-inside-of-you
-#![feature(core_intrinsics, pointer_byte_offsets)]
+#![feature(core_intrinsics)]
```

Comment:
fyi: this test does not use either of those features :/

Reply:
Ah, that's probably a silly copy-paste error then. Can you remove both feature flags while you are at it?